bottomline_recordandreplay/README.md

```diff
@@ -0,0 +1,53 @@
+# Bottomline Mainframe Record and Replay
+
+## Overview
+
+Bottomline's Mainframe Record and Replay solution is non-invasive in its capability to monitor 3270/5250 users via network traffic.  Bottomline helps customers monitor users and systems in ways more advanced than traditional approaches with our industry leading Fraud and Financial Crime solutions.
+
+This integration is for existing Bottomline and Datadog customers, creates a logger in Bottomline's Record and Replay using a pre-built JSON message format, which can then be sent to Datadog using the standard agent. 
+
+### Monitoring Capabilities
+
+1. Mainframe Users: Record and Replay the users session and log information about what the user did in the session.
+2. Mainframe: User Response Time
+3. Mainframe: Resource Response Time 
```

Comment:
```suggestion
Bottomline's Mainframe Record and Replay solution is non-invasive in its capability to monitor 3270/5250 users via network traffic to help customers monitor users and systems.

With this integration, you can monitor Bottomline's' Record and Replay sessions in Datadog to provide visibility into resource utilization, resource performance, user activity, security events, and system monitors. Customers can also directly access user session replays through Datadog.
```

Reply:
udpated.

Reply:
Hi @nbk96f1 it looks like the monitor section wasn't removed. As a quick note (you may already know this), you can commit the suggestions we make so it updates our suggestions automatically instead of having to manually type them out! :)

Reply:
yes - for some reason this one particular instance wouldn't let me use the submit suggestion.  :)